pom.xml

```diff
@@ -23,7 +23,7 @@
         <dependency.pdfbox.version>2.0.22</dependency.pdfbox.version>
         <dependency.alfresco-jodconverter-core.version>3.0.1.12</dependency.alfresco-jodconverter-core.version>
         <env.project_version>${project.version}</env.project_version>
-        <dependency.alfresco-transform-model.version>1.0.2.12</dependency.alfresco-transform-model.version>
+        <dependency.alfresco-transform-model.version>1.3.0</dependency.alfresco-transform-model.version>
```

Comment:
Did you mean 1.0.3 ? 

But I guess a lib we can live with 1.3.0 ?

Reply:
well, I just followed your last comment here: https://github.com/Alfresco/alfresco-transform-model/pull/77#discussion_r578467111, where you suggest 1.3.0?!